### ISSUE TITLE
prevent publishing WP Report before scheduled end of WP?

### ISSUE THREAD

**Issue by mscholtz** [washingtontrails/vms#262]
VMS just allowed me to create and publish the summary report for a WP even though the WP end date isn't until tomorrow.  I could envision wanting validation here to prevent over-eager leaders from jumping the gun, or simply misclicking.  @WaBirder57 your call if this is important.

**Comment by WaBirder57** [comment]
Yes, this would be nice to have.  Medium priority fix to add the date check to prevent publishing a summary report until after the end date of the WP.

**Comment by davisagli** [comment]
It now validates against the end date and won't let you change the status to Publish if today is before the end date.
